packages/app/src/models/history.ts

```diff
@@ -21,8 +21,6 @@ import store from '@kui-shell/core/models/store'
 const key = 'openwhisk.history'
 
 export interface HistoryLine {
-  entityType?: string
-  verb?: string
```

Comment:
The usage of entityType and verb is removed by this PR. We may want to re-introduce them for other purposes, but removing them in this PR makes errors easier to be caught.